changelogs/1.25.0.yaml

```diff
@@ -314,3 +314,9 @@ new_features:
 - area: build
   change: |
     added compile-time option ``--define=static_extension_registration=disabled`` to disable the automatic static registration of extension factories.
+
+deprecated:
+- area: ext_authz
+  change: |
+    deprecated :ref:`allowed_headers <envoy_v3_api_field_extensions.filters.http.ext_authz.v3.AuthorizationRequest.allowed_headers>` in favour
```

Comment:
```suggestion
    deprecated (1.25.0) :ref:`allowed_headers <envoy_v3_api_field_extensions.filters.http.ext_authz.v3.AuthorizationRequest.allowed_headers>` in favour
```

Reply:
also - i think this needs to be shifted to `current.yaml` - adding here it will never be built - if we add in `main` ill backport to the imminent 1.25 patch release